clang/include/clang/Basic/CodeGenOptions.def

```diff
@@ -128,6 +128,9 @@ CODEGENOPT(XRayAlwaysEmitTypedEvents , 1, 0)
 ///< Set when -fxray-ignore-loops is enabled.
 CODEGENOPT(XRayIgnoreLoops , 1, 0)
 
+///< Set when -fbundle-no-offload-arch is enabled.
+CODEGENOPT(BundleNoOffloadArch, 1, 0) 
```

Comment:
Where is this option being used after being set? I see the driver sets it but from a quick look I don't see where you are checking/using it.

Reply:
I am using it here https://github.com/intel/llvm/pull/6482/files#diff-de74256bb2429b219330d496829290947aca7ba4c88d0db1c548f7f976352447R39 so that libdevice bundles contain generic NVPTX bc. Will add testt as soon as possible